src/test/java/de/retest/recheck/report/TestReportFilterTest.java

```diff
@@ -177,6 +178,22 @@ void test_replay_result_should_be_filtered_properly() throws Exception {
 		assertThat( differences ).containsExactly( notFilterMe );
 	}
 
+	@Test
+	void test_replay_result_should_keep_golden_master_exceptions() {
+		final NoGoldenMasterActionReplayResult r1 = mock( NoGoldenMasterActionReplayResult.class );
```

Comment:
Find it a bit confusing that replay results are named here `r`-something, whereas below `result` and `filter`. How about:

* `r1` => `noGoldenMasterActionResult`
* `r2` => `stateDiffActionResult`
* `result` => `testResult`
* `filter` => `filteredTestResult`